### ISSUE TITLE
Keyboard disappears after select option is chosen

### ISSUE THREAD

**Issue by nickw** [meteoric/meteor-ionic#97]
It's not clear if the keyboard actually dismisses, or just disappears, but if you select an option in a <select> element, the keyboard goes away and leaves a large blank area at the bottom of the screen.

**Comment by wrannaman** [comment]
@nickw are you also getting the keyboard covering the input field if they are near the bottom of the page?

**Comment by nickw** [comment]
@wrannaman No, I haven't seen that yet. Will try to test soon.

**Comment by aaronthorp** [comment]
yeah definately a bug, having major dramas with keyboard and focusing on input elements at the moment :(

**Comment by guilhermedecampo** [comment]
Any news on this? I'm having the error. :disappointed:

**Comment by aaronthorp** [comment]
Here is a video demo of the frustration, it seems according to some `console.log` output that the select gets focus and fires the `focus` event but more often than not immediately fires the `blur` event for the same element, not sure why or how, maybe that helps with the debugging?

**Versions:** iOS 8.3, Meteor 1.1.0.2

https://dl.dropboxusercontent.com/u/106020176/MeteoricFrustrating.m4v

**Comment by aaronthorp** [comment]
Quick update to get it temporarily working for anyone who needs it, in the blur event, add the following to refire the `focus` event on the current element if it closed quickly:

```
if (showTime < 500)
  $(e.currentTarget).focus();
```
